src/tools/clippy/clippy_lints/src/methods/mod.rs

```diff
@@ -3379,7 +3381,7 @@ impl<'tcx> LateLintPass<'tcx> for Methods {
 impl Methods {
     #[allow(clippy::too_many_lines)]
     fn check_methods<'tcx>(&self, cx: &LateContext<'tcx>, expr: &'tcx Expr<'_>) {
-        if let Some((name, [recv, args @ ..], span)) = method_call(expr) {
+        if let Some((name, recv, [args @ ..], span)) = method_call(expr) {
```

Comment:
```suggestion
        if let Some((name, recv, args, span)) = method_call(expr) {
```
There should be a lint for this...